db/migrate/20200108230110_create_teams.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTeams < ActiveRecord::Migration[5.2]
+  def change
+    create_table :teams do |t|
+      t.string :name, limit: 20, null: false
```

Comment:
Maybe the `round_duration` should be a team setting? Not really sure about this or if we want to have a `settings` table.

Reply:
I think `settings` as attributes on the `Team` are fine, but we shouldn't be making up "functional" details while defining only models and associations.